### PR TITLE
Fixes the wrong positioning

### PR DESCRIPTION
**Pull request by thagikura**
On the condition flexWrap = No wrap and justifyContent = center.
This happened because when flexWrap = no wrap, the anchor position
retrieved from the existing view was wrong. Because the OrientationHelper
is changed to perpendicularly when flexWrap is nowrap.

### CODE REVIEW

File: flexbox/src/main/java/com/google/android/flexbox/FlexboxLayoutManager.java [hunk 1]
```diff
@@ -2873,17 +2882,38 @@ private void assignFromView(View anchor) {
                 // We need to use the anchor view as starting from right if the flex direction is
                 // (column or column_reverse) and layout direction is RTL.
                 if (mLayoutFromEnd) {
-                    mCoordinate = mOrientationHelper.getDecoratedStart(anchor) +
-                            mOrientationHelper.getTotalSpaceChange();
+                    // When the flex wrap is nowrap, the orientation helper is changed to
```
Comment (MGaetan89) [inline flexbox/src/main/java/com/google/android/flexbox/FlexboxLayoutManager.java]: Why not add something like this at the beginning of the method:
```java
OrientationHelper orientationHelper;
// When the flex wrap is nowrap, the orientation helper is changed to
// perpendicular. Need to use the mSubOrientationHelper to be consistent with
// when flex wrap is not nowrap.
if (mFlexWrap == FlexWrap.NOWRAP) {
    orientationHelper = mSubOrientationHelper;
} else {
    orientationHelper = mOrientationHelper;
}
```

And then using directly `orientationHelper` in the four changes you made in this method?

Comment (thagikura) [reply flexbox/src/main/java/com/google/android/flexbox/FlexboxLayoutManager.java]: It looks better. Thanks.

**Comment by dhanaforu** [comment]
@thagikura @MGaetan89 Is this fix available in v1.1.0 or do we have next version?

**Comment by MGaetan89** [comment]
Version 1.1.0 was released Sept. 25th, and this was merged Nov. 16th.
So it is not yet included in any release.

**Comment by dhanaforu** [comment]
Thanks @MGaetan89.

**Pull request merged**